kube/app/deployment.yml

```diff
@@ -73,6 +73,33 @@ spec:
                 secretKeyRef:
                   name: session-secret
                   key: session-secret
+            - name: FILE_VAULT_CLIENT_SECRET
+              valueFrom:
+                secretKeyRef:
+                  name: keycloak-client
+                  key: secret
+            - name: FILE_VAULT_CLIENT_ID
+              valueFrom:
+                secretKeyRef:
+                  name: keycloak-client
+                  key: id
+            - name: FILE_VAULT_USERNAME
+              valueFrom:
+                secretKeyRef:
+                  name: keycloak-user
+                  key: username
+            - name: FILE_VAULT_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: keycloak-user
+                  key: password
+            - name: FILE_VAULT_URL
+            {{ if eq .KUBE_NAMESPACE .UAT_ENV }}
+              value: https://fv-{{ .APP_NAME }}.uat.sas-notprod.homeoffice.gov.uk/file
+            {{ else if eq .KUBE_NAMESPACE .BRANCH_ENV }}
+              value: https://fv-{{ .DRONE_SOURCE_BRANCH }}.branch.sas-notprod.homeoffice.gov.uk/file
+            {{ end }}
```

Comment:
the values for STG and PROD environment are missing

Reply:
These will be added on a separate ticket